engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -4,6 +4,69 @@
 
 #include "flutter/shell/platform/embedder/embedder_semantics_update.h"
 
+namespace {
+
+// TODO(hangyujin): Update these two functions once the SemanticsFlags in
+// dart:ui are updated to use tristate/quad-state flags for properties like
+// isChecked and isSelected.
+FlutterCheckState ToFlutterCheckState(bool isNotNone,
+                                      bool isTrue,
+                                      bool isMixed) {
```

Comment:
Nit: please use snake casing here in C++

```suggestion
FlutterCheckState ToFlutterCheckState(bool is_not_none,
                                      bool is_true,
                                      bool is_mixed) {
```

Reply:
Same comment for `ToFlutterTristate` below